From: =?utf-8?q?Florian_M=C3=BCllner?= <fmuellner@gnome.org>
Date: Fri, 31 Mar 2017 16:59:38 +0200
Subject: appchooserdialog: Fix CSS loading

load_from_path() doesn't support resource:// URIs, so we're currently
trying to load a very weirdly named file from the filesystem ...

Origin: upstream, 0.7, commit:6d29fb2a8254dada14f0e8c485120f0bc6e6c0a8
---
 src/appchooserdialog.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/appchooserdialog.c b/src/appchooserdialog.c
index fc526ae..467f9fe 100644
--- a/src/appchooserdialog.c
+++ b/src/appchooserdialog.c
@@ -120,7 +120,7 @@ app_chooser_dialog_new (const char **choices,
   if (provider == NULL)
     {
       provider = gtk_css_provider_new ();
-      gtk_css_provider_load_from_path (provider, "resource:///org/freedesktop/portal/desktop/gtk/appchooserdialog.css", NULL);
+      gtk_css_provider_load_from_resource (provider, "/org/freedesktop/portal/desktop/gtk/appchooserdialog.css");
       gtk_style_context_add_provider_for_screen (gdk_screen_get_default (),
                                                  GTK_STYLE_PROVIDER (provider),
                                                  GTK_STYLE_PROVIDER_PRIORITY_APPLICATION);
